src/format/scan.rs

```diff
@@ -243,13 +243,26 @@ pub(super) fn timezone_offset<F>(s: &str, consume_colon: F) -> ParseResult<(&str
 where
     F: FnMut(&str) -> ParseResult<&str>,
 {
-    timezone_offset_internal(s, consume_colon, false)
+    timezone_offset_internal(s, consume_colon, false, true)
 }
 
+/// The `consume_colon` function is used to parse a mandatory or optional `:`
```

Comment:
Thanks for adding comments here.

I'd like to have the first line be a concise (one line) explanation of what this function does, before the comment goes into detail on what arguments mean.